.github/workflows/release-cli.yml

```
@@ -91,6 +153,10 @@ jobs:
         with:
           path: dist/cli
           merge-multiple: true
+      - name: Generate checksums
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:2:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)